fdbcli/ChangeFeedCommand.actor.cpp

```diff
@@ -41,11 +41,12 @@ ACTOR Future<Void> changeFeedList(Database db) {
 		try {
 			tr.setOption(FDBTransactionOptions::ACCESS_SYSTEM_KEYS);
 			tr.setOption(FDBTransactionOptions::PRIORITY_SYSTEM_IMMEDIATE);
+			tr.setOption(FDBTransactionOptions::RAW_ACCESS);
```

Comment:
There's no problem setting this here, but just as a note `RAW_ACCESS` is implied by `ACCESS_SYSTEM_KEYS`